src/main/java/it/pagopa/pm/gateway/service/XpayService.java

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.pm.gateway.service;
+
+import it.pagopa.pm.gateway.dto.xpay.AuthPaymentXPayRequest;
+import it.pagopa.pm.gateway.dto.xpay.AuthPaymentXPayResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class XpayService {
+
+    @Value("${xpay.authenticationUrl}")
+    private String XPAY_AUTH_URL;
+
+    @Autowired
+    RestTemplate xpayRestTemplate;
+
+    public AuthPaymentXPayResponse postForObject(AuthPaymentXPayRequest xPayRequest) {
```

Comment:
postForObject is a too generic name, better rename this method in something more self-explaining for the purpose of its invocation